src/docs/sdk/event-payloads/properties/measurements.mdx

```diff
@@ -0,0 +1,18 @@
+`measurements`
+
+: _Optional_. An object containing standard/custom measurements with keys signifying the name of the measurement.
+
+Standard measurement keys currently supported are from the set  `"fp" | "lcp" | "fid" | "ttfb" | "ttfb.requesttime"`.
```

Comment:
Can we clarify what "supported" means here?

Also, what do you think about linking to the relay/sentry allow list code here? 

This is all browser web vitals, we also need to include the mobile vitals here as well.

Reply:
I think 'supported' is open to discussion based on either
* what is currently implemented end-end in browser/mobile
* what gets assigned units automatically if we say we _need_ units for custom measurements from the user (?)